### PR TITLE
bigtable: column family support when creating tables

### PR DESCRIPTION
**Pull request by callmehiphop**
Relates to https://github.com/GoogleCloudPlatform/gcloud-node/issues/1451

This adds support for specifying column families when creating a table.

### CODE REVIEW

File: test/bigtable/index.js [hunk 1]
```diff
@@ -219,6 +222,44 @@ describe('Bigtable', function() {
       bigtable.createTable(TABLE_ID, options, assert.ifError);
     });
 
+    it('should set the column families', function(done) {
```
Comment (stephenplusplus) [inline test/bigtable/index.js]: Can you create these tests:

- describe('options.families', ...
  - it should accept a string family name
  - it should accept a string garbage collection rule
  - it should accept an object garbage collection rule

The reason being, if someone makes a change to your test that currently combines these, they might accidentally remove one of the checks. Having individual tests that can fail will reduce that chance.

Comment (callmehiphop) [inline test/bigtable/index.js]: Done! PTAL

**Comment by stephenplusplus** [comment]
Cool, LGTM.

**Comment by stephenplusplus** [comment]
Rebased and merged in: https://github.com/GoogleCloudPlatform/gcloud-node/commit/1a42246accae61d0ce114c9f65610311ab229dae

**Pull request closed without merging**